addon/adapters/application.js

```diff
@@ -0,0 +1,181 @@
+import Ember from 'ember';
+import DS from 'ember-data';
+
+export default DS.RESTAdapter.extend({
+
+    defaultSerializer: '-parse',
```

Comment:
It would be ideal to not reformat all the files from two spaces to tabs or four spaces :-/ It breaks the history and git blame, and I'm not sold on a justification for overhauling formatting.